### ISSUE TITLE
Support for cytube.

### ISSUE THREAD

**Issue by Azunyan-** [ajayyy/SponsorBlock#150]
I'm mostly asking about support for their site: https://cytu.be/ altough they are open source and one could add support via their backend and make a fork.
I'm guessing the leader of the room is going to have the addon installed since it is the leader that is in charge of how to video is synced.

**Comment by ajayyy** [comment]
This site uses YouTube embeds right? If so, it should work.

**Comment by Azunyan-** [comment]
Yeah it uses YouTube embeds, it hasn't worked for me before but does now.
I just doubled checked on a video that does have sponsorship that are marked on YouTube (LTT).

Queueing the video and me being the leader does skip it, but if I queue another video with sponsorship in it will retain the same skip points from the past video and incorrectly skip in the second video.

**Comment by ajayyy** [comment]
Oh sorry, I forgot about that issue. Someone else has mentioned it before. The issue is that it currently looks at the URL of the frame, which remains constant even when the video changes. So, it only works for the first video. This one is a little harder to fix.

**Comment by ajayyy** [comment]
`durationchange` listener

**Comment by ajayyy** [comment]
Cytube seems to just be resetting the timstamp anyway. I don't thinking fixing it showing the wrong sponsors will help.

**Comment by Azunyan-** [comment]
You need to be the room leader to make sure your local change of the current time in the video is the one the room follows and it will keep it from resetting. This means that only the room leader needs the add-on installed.

**Comment by ajayyy** [comment]
I was the room leader and couldn't get that to happen.

**Comment by Azunyan-** [comment]
I just tried it and it worked for me in a room with only me in it.
The video successfully skipped the first two "ads/sponsorships" in the video and continued playing without resetting to the prior point before the ads.
I was the room leader.
The room is registered to my account on cytube.
I'm running Firefox 71.0 (64-bit)
SponsorBlock Version 1.2.13 .
Video I used to test this: https://www.youtube.com/watch?v=4DKLA7w9eeA

I can't think of anything else that might be relevant for you to try and replicate this but do ask if you want more info or a .gif of me skipping an ad.
